doc-surrealist_versioned_docs/version-2.x/tutorials/How-to-define-a-schema-in-Surrealist.mdx

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 2
+sidebar_label: Defining A Schema In Surrealist
+title: Surrealist | Embedding Surrealist  
+description: Surrealist is a user interface for interacting with your SurrealDB database visually. It enables you to seamlessly connect to any SurrealDB instance, allowing you to execute queries, explore your tables, design your schemas, and much more.
```

Comment:
Small nit: "enables you" is quickly followed by "allowing you". How about this? (Feel free to ignore, it's a very small point)

Surrealist seamlessly connects to any SurrealDB instance, allowing you to execute queries, explore your tables, design your schemas, and much more.